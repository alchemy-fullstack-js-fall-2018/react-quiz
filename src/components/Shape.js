import React from 'react';

export default function ColorSelector({ fontColor, backgroundColor, text }) {
  return (
    <div style={{ background: backgroundColor, color: fontColor }}>{text}</div>
  );
}
