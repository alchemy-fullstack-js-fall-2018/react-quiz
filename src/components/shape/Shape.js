import React, { Fragment } from 'react';

const Shape = ({ text, textChange, colorChange }) => {
  return (
    <Fragment>
      <div>
        <span value={text} onChange={textChange} onColorChange={colorChange}></span>
      </div>
    </Fragment>
  );
};

export default Shape;
