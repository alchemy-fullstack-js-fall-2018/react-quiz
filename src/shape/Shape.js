import TextSelector from '../textSelector/TextSelector'
