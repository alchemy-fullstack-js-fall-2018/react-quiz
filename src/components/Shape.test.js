import React from 'react';
import Shape from './Shape';
import render from 'react-test-renderer';

describe('Shape Component Test', () => {
  test('Shape snapshot', () => {
    expect(render
      .create(<Shape />)
      .toJSON()).toMatchSnapshot();
  });
});
