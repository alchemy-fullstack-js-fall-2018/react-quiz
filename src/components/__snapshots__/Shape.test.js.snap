// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shape Component Test Shape snapshot 1`] = `
<div
  style={
    Object {
      "background": undefined,
      "color": undefined,
    }
  }
/>
`;
